plugins/techdocs-node/src/stages/generate/helpers.ts

```diff
@@ -101,9 +101,12 @@ export const getRepoUrlFromLocationAnnotation = (
   if (locationType === 'url') {
     const integration = scmIntegrations.byUrl(target);
 
-    // We only support it for github and gitlab for now as the edit_uri
+    // We only support it for github, gitlab and bitbucketServer for now as the edit_uri
     // is not properly supported for others yet.
-    if (integration && ['github', 'gitlab'].includes(integration.type)) {
+    if (
+      integration &&
+      ['github', 'gitlab', 'bitbucketServer'].includes(integration.type)
```

Comment:
Bitbucket Cloud supports edit URLs, too, but out of scope here.

Maybe I pick that up for Bitbucket Cloud.